src/client.js

```diff
@@ -183,6 +184,10 @@ const Client = IgeClass.extend({
 
 		promise.then((game) => {
 			ige.game.data = game.data;
+			// let's try here
+			ige.addComponent(IgeInitPixi);
+			ige.entityTrack = new EntityTrack();
+			ige.entityTrack.applyRendererEvents();
```

Comment:
would be cleaner if `applyRendererEvents` method was invoked at the end of `EntityTrack` constructor instead of here.